bluebottle/activities/models.py

```diff
@@ -170,9 +170,9 @@ def save(self, *args, **kwargs):
 
     def __unicode__(self):
         if self.user:
-            return unicode(_("Organizer: {name}".format(name=self.user.full_name)))
+            return self.user.full_name
         else:
-            return _('Organizer')
```

Comment:
This is here mainly for documentation purposes. If there is an organizer, it would always have a user. We might want to change how we make sure documentation prints nicely but now side effects might render like "Send message to None"

Reply:
Guess you fixed this with the templates...?